qt/widgets/common/src/PropertyHandler.cpp

```diff
@@ -1079,23 +1080,49 @@ void PropertyHandler::fix(const QString &parName) {
 }
 
 /**
- * Remove the tie.
- * @param prop :: The tie property to remove
- */
+* Remove the tie.
+* @param prop :: The tie property to remove
+* @param globalName :: Name of the parameter in compoite function
+* (e.g. f1.omega)
+*/
+void PropertyHandler::removeTie(QtProperty *prop, std::string globalName) {
+	QString parName = m_ties.key(prop, "");
+	if (parName.isEmpty())
+		return;
+
+	QtProperty *parProp = getParameterProperty(parName);
+	if (parProp) {
+		m_browser->m_changeSlotsEnabled = false;
+		auto tom = parName.toStdString();
+		auto &cfun = *m_browser->compositeFunction();
```

Comment:
Again do we need the abbreviation on `parProp` and `cfun`? What does `tom` mean, I can't see it being used?
The same applies to `PropertyHandler::removeTie` below.